src/ipaperftest/plugins/authenticationtest.py

```diff
@@ -180,6 +181,11 @@ def run(self, ctx):
                          error="Failed to convert host-find output to int. "
                                "Value was: %s" % host_find_output)
 
+        self.run_ansible_playbook_from_template(
```

Comment:
Shouldn't this run only when `disable_selinux` is passed?

Reply:
Yes it should, nice catch.